### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by rodrigoWolox**
## Summary
Responsive Design

## Screenshot
resposive2

## Trello Card
https://trello.com/c/U1yT8q57/15-dise%C3%B1o-responsive

### CODE REVIEW

File: scss/footer.scss [hunk 1]
```diff
@@ -24,3 +24,26 @@
...
+  }
+
+  .copyrights {
+    align-self: center;
+    font-size: 6px;
+  }
+  
+  .footer-logo {
+    margin-bottom: 40px;
+    width: 70%;
```
Comment (mmamaniwolox) [inline scss/footer.scss]: Use fixed value, percentages may lead to undesired sizes

File: scss/nav.scss [hunk 2]
```diff
@@ -18,3 +18,14 @@
...
   height: 30px;
 }
+
+@media only screen and (max-width: 1024px) {
+  .nav-logo {
+    height: 25px;
+  }
+
+  .nav-bar {
+    padding: 10px 10px;
```
Comment (mmamaniwolox) [inline scss/nav.scss]: padding: 10px

File: scss/tech.scss [hunk 3]
```diff
@@ -51,3 +51,22 @@
...
+  .tech-container {
+    border-top: 2px solid $earls-green;
+    border-radius: 0px;
+    
+  }
+
+  .head-tech {
+    border: none;
+  }
+}
```
Comment (mmamaniwolox) [inline scss/tech.scss]: Add tailing line

File: scss/titles.scss [hunk 4]
```diff
@@ -18,4 +18,20 @@
...
   } 
-}
+}  
+
+@media only screen and (max-width: 1024px) {
+  .header-title {
+    display: none;
+  }
+
+  .page-title {
```
Comment (mmamaniwolox) [inline scss/titles.scss]: `.page-title::before { ...} `
Do you really need to override all those attributes?

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

**Review by mmamaniwolox — COMMENTED** [review]
Footer is too tall and copyright its not readable

File: scss/titles.scss (hunk 4 above)
Comment (rodrigoWolox) [reply scss/titles.scss]: Not really. Just did it to achieve the card's layout image.

**Review by mmamaniwolox — APPROVED** [review]

**Pull request merged**